frontend/src/main/resources/abs/lang/abslang.abs

```diff
@@ -616,6 +616,9 @@ data Annotation = TypeAnnotation;
  
 [TypeAnnotation]
 data LocationType = Far | Near | Somewhere | Infer;
+
+[TypeAnnotation]
+data NullableType = NonNull | Nullable;
```

Comment:
We steal notation from Java unless it's inconvenient :-)  JSR-305 seems to use `Nonnull` for this annotation, not `NonNull`; should we do the same?

Reply:
Well, I am in favor of `NonNull`, as I find it clearer, but I could change it